sdk/ml/azure-ai-ml/azure/ai/ml/operations/_schedule_operations.py

```diff
@@ -4,7 +4,7 @@
 # pylint: disable=protected-access
 from typing import Any, Iterable
 
-from azure.ai.ml._restclient.v2022_10_01 import AzureMachineLearningWorkspaces as ServiceClient102022
+from azure.ai.ml._restclient.v2023_02_01_preview import AzureMachineLearningWorkspaces as ServiceClient022023Preview
```

Comment:
Why did you need to change this?

Reply:
schedule command e2e job operations were failing without this change since we're trying to read queue settings from the rest object